iterativeTrim.py

```diff
@@ -168,6 +168,12 @@ def writeChConfig(chConfig,vfat,vfatID,trimDAC,trimPol,mask,maskReason):
     parser.add_argument("-d","--debug", action="store_true", help="Prints additional debugging information")
     parser.add_argument("-l","--latency", type=int, help="CFG_LATENCY value to be used",default=33)
     parser.add_argument("-m","--maxIter", type=int, help="Maximum number of iterations to perform (e.g. number of scurves to take)", default=4)
+    
+    parser.add_argument("--sigmaOffset", type=int, help="Will align the mean - sigmaOffset*sigma", default=0)
+    parser.add_argument("--highTrimCutoff", type=int, help="Will weight channels that have a trim value above this (when set to 63, has no effect)", default=63)
+    parser.add_argument("--highTrimWeight", type=int, help="Will apply this weight to channels that have a trim value above the cutoff", default=50)    
+    parser.add_argument("--highNoiseCut", type=float, default=1.5, help="Threshold in fC for masking the channel due to high noise")
+    
```

Comment:
I have this itching on the back of my neck that makes me wonder whether these types of settings are better stored in some config file that is read by the trim routine, rather than passed around as arguments?

Reply:
If it grows to more than 4, I think we can make a config file. I have moved the default values to a separate file, like [anaInfo.py](https://github.com/cms-gem-daq-project/gem-plotting-tools/blob/develop/utils/anaInfo.py) for gem-plotting-tools.